tests/unit/s2n_client_hello_recv_test.c

```diff
@@ -75,13 +75,12 @@ int main(int argc, char **argv)
 
     /* Test we can successfully receive an sslv2 client hello and set a
      * tls12 connection */
-    for (uint8_t i = 0; i < 2; i++) {
-        if (i == 1) {
-            EXPECT_SUCCESS(s2n_enable_tls13_in_test());
-        }
+    {
+        /* "enable" tls13, to test under default s2n-tls behavior */
+        EXPECT_SUCCESS(s2n_enable_tls13_in_test());
```

Comment:
For the default behavior, you actually want s2n_reset_tls13_in_test. "enable" sets the default config to "default_tls13", which isn't the default s2n-tls behavior.